echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/jira/JiraNotificationService.java

```diff
@@ -189,10 +189,9 @@ private Map<String, Object> issueRequestBody(Notification notification) {
   }
 
   private Map errors(Exception exception) {
-    if (exception instanceof RetrofitError) {
+    if (exception instanceof SpinnakerHttpException) {
       try {
-        return mapper.readValue(
-            ((RetrofitError) exception).getResponse().getBody().in(), Map.class);
+        return ((SpinnakerHttpException) exception).getResponseBody();
       } catch (Exception e) {
```

Comment:
Do we still need the catch block here, since SpinnakerHttpException.getResponseBody is a plain old accessor?